tpm2/Makefile

```diff
@@ -23,36 +23,36 @@ LLIBS += -ltss2-rc#                      TPM 2.0 Return Code Utilities
 LLIBS += -lssl#                          OpenSSL
 LLIBS += -lcrypto#                       libcrypto
 LLIBS += -lkmip#                         libkmip
-LLIBS += -lkmythlogger#                  kmyth logging utility shared library
+LLIBS += -lkmyth#                        kmyth shared library
```

Comment:
This is kind of a trivial comment, but I was looking at the GNU make manual for something else and noticed that the convention is to use LDLIBS and LDFLAGS (as opposed to LLIBS and LFLAGS). LFLAGS is actually the variable used to pass parameters to Lex.